package.json

```diff
@@ -35,16 +42,19 @@
     "grunt-git": "^1.0.0",
     "grunt-karma": "^2.0.0",
     "grunt-mocha-test": "^0.12.7",
+    "husky": "^0.13.4",
     "injectr": "0.5.1",
     "jasmine-core": "^2.5.1",
     "karma": "^1.3.0",
     "karma-jasmine": "^1.0.2",
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-sauce-launcher": "^1.0.0",
     "karma-sinon": "^1.0.5",
+    "lint-staged": "^3.6.0",
     "load-grunt-tasks": "^3.5.2",
     "mocha": "^3.0.2",
     "phantomjs-prebuilt": "^2.1.12",
+    "prettier-eslint-cli": "^4.0.4",
```

Comment:
I think make sense to remove it (also help a little toward moving to a [gruntless world](https://github.com/opentable/oc/issues/464)).

You could now run `prettier-eslint` instead if you want for developement